json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -299,6 +299,14 @@ public function build_current_site_response( ) {
 				if ( $is_jetpack ) {
 					$response['options']['jetpack_version'] = get_option( 'jetpack_version' );
 
+					// If we are not on WPCOM, force WordPress to recalculate available updates.
+					if ( ! IS_WPCOM ) {
```

Comment:
Should we also do a check for is main site her? Since we would skip this part if we are not dealing with the main site would it still return these results?